### PR TITLE
[Kotlin CleanUp ] CardTemplatePreviewerTest.kt,DeckPickerCheckDataBaseListener.kt

### PR DESCRIPTION
**Pull request by Prince-kushwaha**
## Pull Request template
Kotlin CleanUp of CardTemplatePreviewerTest.kt,DeckPickerCheckDataBaseListener.kt

### CODE REVIEW

**Review by lukstbit — APPROVED** [review]
Thanks!

File: AnkiDroid/src/test/java/com/ichi2/anki/DeckPickerCheckDatabaseListenerTest.kt [hunk 1]
```diff
@@ -152,18 +149,15 @@ class DeckPickerCheckDatabaseListenerTest : RobolectricTest() {
...
-            return mDidDisplayMessage
-        }
+        val didDisplayMessage: Boolean
+            get() = mDidDisplayMessage
 
-        fun didDisplayLockedDialog(): Boolean {
-            return mDidDisplayDbLocked
-        }
+        val didDisplayLockedDialog: Boolean
+            get() = mDidDisplayDbLocked
```
[the remark this answers is not in the document]
Comment (Prince-kushwaha) [reply AnkiDroid/src/test/java/com/ichi2/anki/DeckPickerCheckDatabaseListenerTest.kt:161]: @oakkitten In kotlinCleanUp i have to  replace getters with variables

[the remark this answers is not in the document]
Comment (BrayanDSO) [reply AnkiDroid/src/test/java/com/ichi2/anki/DeckPickerCheckDatabaseListenerTest.kt:161]: Agreed here with oakkitten.

File: AnkiDroid/src/test/java/com/ichi2/anki/DeckPickerCheckDatabaseListenerTest.kt [hunk 2]
```diff
@@ -127,43 +126,41 @@ class DeckPickerCheckDatabaseListenerTest : RobolectricTest() {
...
-        private var mDidDisplayDialogLoadFailed = false
-        private var mDidDisplayMessage = false
-        private var mDidDisplayDbLocked = false
-        fun didDisplayDialogLoadFailed(): Boolean {
-            return mDidDisplayDialogLoadFailed
-        }
+        var didDisplayDialogLoadFailed = false
+            private set(value) {
+                field = value
+            }
```
Comment (BrayanDSO) [inline AnkiDroid/src/test/java/com/ichi2/anki/DeckPickerCheckDatabaseListenerTest.kt:133]: Can this and the below properties be simplified like this?
```suggestion
        private var didDisplayDialogLoadFailed = false
```

Comment (Prince-kushwaha) [reply AnkiDroid/src/test/java/com/ichi2/anki/DeckPickerCheckDatabaseListenerTest.kt:133]: i already simplified the below properties

Comment (oakkitten) [reply AnkiDroid/src/test/java/com/ichi2/anki/DeckPickerCheckDatabaseListenerTest.kt:133]: Just a note, this
```kt
var didDisplayDialogLoadFailed = false
    private set(value) {
        field = value
    }
```
is the same as
```kt
var didDisplayDialogLoadFailed = false
    private set
```

Comment (Prince-kushwaha) [reply AnkiDroid/src/test/java/com/ichi2/anki/DeckPickerCheckDatabaseListenerTest.kt]: ok

**Review by BrayanDSO — APPROVED** [review]

**Review by lukstbit — APPROVED** [review]

**Pull request merged**